packages/app/src/cli/services/context.ts

```diff
@@ -816,6 +816,7 @@ interface ReusedValuesOptions {
  */
 function showReusedDevValues({organization, selectedApp, selectedStore, cachedInfo}: ReusedValuesOptions) {
   if (!cachedInfo) return
+  if (sniffForJson()) return
```

Comment:
This ensures that it doesn't display any info boxes when the `--json` flag is in use, to avoid breaking JSON parsing libraries like `jq`